code/modules/client/preferences_ui.dm

```diff
@@ -630,6 +630,9 @@
 			key_bindings[full_key] = sortList(key_bindings[full_key])
 			current_client.update_movement_keys()
 			save_keybinds()
+			// If they have a mob
+			if(user)
+				SEND_SIGNAL(user, COMSIG_MOB_KEYBINDINGS_UPDATED, GLOB.keybindings_by_name[kb_name])
```

Comment:
I think sendsignal checks for null, check that though

Reply:
It only checks if it has a comp_lookup or sig_type registered ,but not wheter the object its called onto is null.